Automattic/jetpack
# Social: Update admin pricing page

Fixes https://github.com/Automattic/jpop-issues/issues/8502

In the Social admin page there were some items that weren't updated as the features were added. This small page fixes that

## Proposed changes:

* Moved video sharing to included on the Advanced plan
* Moves SIG as included in the Advanced plan

### Other information:


## Jetpack product discussion


https://github.com/Automattic/jpop-issues/issues/8502
## Does this pull request change what data or activity we track or use?


No
## Testing instructions:






* In `projects/plugins/social/src/js/components/admin-page/index.jsx` change the initial value of `forceDisplayPricingPage` to `true`.
* Go to the Social admin page, and check the pricing page, which should look like this:
<img width="1268" alt="CleanShot 2023-09-19 at 13 22 18 png" src="https://github.com/Automattic/jetpack/assets/36671565/934f0cb5-9e4f-4311-83b2-77ca64ef7d54">
